packages/valory/skills/decision_maker_abci/behaviours/base.py

```diff
@@ -618,6 +624,13 @@ def _write_benchmark_results(
             )
             return
 
+        if old_amounts is not None:
+            l0_start, l0_end = old_amounts[0], new_amounts[0]
+            l1_start, l1_end = old_amounts[1], new_amounts[1]
```

Comment:
Let's convert this to a structure, for example, a dataclass called `Liquidity` that has the l0s and l1s.